### DISCUSSION TITLE
Design

### DISCUSSION THREAD

**Discussion by Natashka333** [Together-100Devs/Together#17] [General]
Hey group! What do you think about this design? I would love to get some feedback from you.

## Landing page

![landing Page](https://i.imgur.com/3zeJENZ.png)

## Login Page
![login Page](https://i.ibb.co/51ndkV9/Screen-Shot-2022-11-06-at-8-49-02-PM.png)

## Calendar page
![Calendar Page](https://i.ibb.co/qrVQqw2/Screen-Shot-2022-11-06-at-8-49-15-PM.png)

**Comment by Caleb-Cohen** [comment]
These look great! I'm wondering if it would be better to merge the login with discord onto the landing page. The door is super cute, but I'm not sure if it's intuitive enough for a user.

There is a + mark for every day currently on the calendar, but I do like the concept of Add. I'm concerned it would be difficult to link edit/deleting to specific events from the top right options. 

If you have extra time, I'd love to see how you can visualize event views when you click on them. @intelagense has a discussion called whiteboard visualization #9 for the types of variables that will be viewable for the event view. Instinc dev is working on form input #7, she may want some help with the design.

**Comment by JoeDravarol** [comment]
I also like to door design/feature though like @Caleb-Cohen mentioned, it might not be intuitive for a user. I think, it could work if we only have the door on the screen, so that it's the only thing shown to user, it might be more inuitive this way, seeing the door as the only interactable element? Then moved/shift the about app section under the door so user can scroll to it?

Personally, I think we should have a seperate page for authenticated user, showing all of their created events and they can edit or delete specific events there?

**Comment by Caleb-Cohen** [comment]
> Personally, I think we should have a seperate page for authenticated user, showing all of their created events and they can edit or delete specific events there?

Similar to the profile in binary upload boom displaying previous posts? I like that.

**Comment by yuhwaa** [comment]
Love the design and color scheme. I made a lo-fi visual for the landing with some of the feedback plus mine; maybe something like this? and we can also add a hover-over state to the door so it opens and is more intuitive that you can click on it. 
![lofimockup](https://user-images.githubusercontent.com/101489014/200472818-eefa3998-bc13-4eb7-9fa3-bd05989c170c.jpg)

**Reply by yuhwaa** [reply]
My door might be too big on an actual browser scale, but leaving it here for reference.

**Reply by JoeDravarol** [reply]
I like the hover-over state idea.

**Comment by Natashka333** [comment]
Awesome! Thank you, everyone, for the feedback! @Caleb-Cohen, here is my vision of the Login button on the landing page (I am not giving up on the door, just playing around with your suggestion in Figma). @JoeDravarol, we always can bring back a separate authenticated user page, but let me know what you think about the updated landing page. @yuhwaa, I appreciate your participation. Your idea looks great and we can keep the door. @Caleb-Cohen and @JoeDravarol it is up to you.

![Landing Page](https://i.ibb.co/z7vkq1G/Screen-Shot-2022-11-07-at-11-05-01-PM.png)

**Reply by JoeDravarol** [reply]
This looks great, I really like this.

**Reply by yuhwaa** [reply]
I like this too! it looks more organized and everything is in its own box. thanks for the update! I like that you can log in on the landing right away. We can always add 'more options to sign in' under the button and link out to another page for different auth but the landing signals that discord is the preferred auth method.

**Comment by colinborows** [comment]
Hey all, posted this in the discord, uploading here. Idea for a logo for the app to match Natashka's modern tech aesthetic and color palette. Would love to hear your thoughts! 

<img width="604" alt="Screenshot 2022-11-08 at 5 38 53 PM" src="https://user-images.githubusercontent.com/82629904/200699602-965d3e1c-dabb-4080-b766-7f7dbf63a97f.png">
<img width="672" alt="Screenshot 2022-11-08 at 3 43 01 PM" src="https://user-images.githubusercontent.com/82629904/200699622-33b0d755-9f84-40c9-9a1e-2d4deb587573.png">

**Reply by Caleb-Cohen** [reply]
Love the color change. This looks great. @Natashka333 do you have any thoughts?

**Reply by Natashka333** [reply]
@Caleb-Cohen, I have not only thoughts, but I have made some mockups! @colinborows, I found both logos adorable. Thank you for bringing your creativity to the team! I am attaching landing pages with both logos to pick the best one. Also, I have been working today on the calendar and event page. Your thoughts are always appreciated. 

![logo 1](https://i.ibb.co/q9BmJHV/Screen-Shot-2022-11-08-at-10-44-46-PM.png)
![logo 2](https://i.ibb.co/W3ZTfsB/Screen-Shot-2022-11-08-at-10-42-24-PM.png)
![calendar page](https://i.ibb.co/16bWRjT/Screen-Shot-2022-11-08-at-11-06-02-PM.png)
![add event](https://i.ibb.co/2gHjVWD/Screen-Shot-2022-11-08-at-10-42-46-PM.png)
![add event extra](https://i.ibb.co/tbLqrKW/Screen-Shot-2022-11-08-at-11-10-24-PM.png)

**Reply by Caleb-Cohen** [reply]
Excellent work! Have you considered incorporating @colinborows logo in more places? Like in the event view page in the top right?

**Reply by colinborows** [reply]
Awesome. Added variations of the logo here: https://github.com/colinborows/togetherLogo
Let me know if we need other versions, file types, etc.

A few design notes on the layout:

- Try adding some more negative space in general, current design is hugging the margins a bit tight (e.g. buttons, "Who are we?")
- Use a lighter font weight for the body text to differentiate from the headers

Looking great! Happy to collaborate on any other design elements as well.

**Reply by Natashka333** [reply]
@colinborows thank you for your feedback. I will work on your notes today. Also, I will send the edit&delete mockup later today. Sorry for just now getting back to you, guys.

**Comment by Caleb-Cohen** [comment]
@colinborows can provide just the calendar so we can put that in the favicon? If yes, would you prefer to use color or B&W?

**Reply by intelagense** [reply]
☑️ I vote for the color favicon... the color palette is unique enough to be recognizable with a bunch of open tabs.

**Reply by colinborows** [reply]
I agree with @intelagense. Added favicon to this folder, this .ico works optimized for 16, 32, 48, and 64px. Favicons are a funky format. Should work on all devices! https://github.com/colinborows/togetherLogo/tree/main/favicon

**Reply by Caleb-Cohen** [reply]
Just implemented it. Looks pretttttty sweet!

**Comment by Natashka333** [comment]
@colinborows @Caleb-Cohen @intelagense, here is what I have worked on today:
- adding negative space;
- lighter font-weight;
- adding the logo to add and event forms;
- event form;
- edit form;
- delete form.


![landing page](https://i.ibb.co/sPJbLxz/Screen-Shot-2022-11-13-at-10-17-46-PM.png)
![calendar page](https://i.ibb.co/60wWB8r/Screen-Shot-2022-11-13-at-10-12-57-PM.png)
![add event form](https://i.ibb.co/zmFpby4/Screen-Shot-2022-11-13-at-10-13-22-PM.png)

**Comment by Natashka333** [comment]
![Screen Shot 2022-11-13 at 10 13 53 PM](https://user-images.githubusercontent.com/102708144/201576728-5a704de1-6cc0-4e04-8d71-19e06e47ee8f.png)
![delete form](https://i.ibb.co/9ZWVPxX/Screen-Shot-2022-11-13-at-10-14-20-PM.png)
![delete form](https://i.ibb.co/stgT7xZ/Screen-Shot-2022-11-13-at-10-15-15-PM.png)

**Reply by Caleb-Cohen** [reply]
Can you include the whole diagram? I see lines coming and going, but I'd like to see it all pieces together.

**Reply by ghost** [reply]
are the images above broken into or able to be broke into smaller chunks of their selves, ex(just the button that goes inside the "Delete the event" example, and the X to close button.  I'm partly assuming so because of the implications.

**Reply by ghost** [reply]
Also, nice work it would be extremely cool if we have modular pieces.

**Comment by Natashka333** [comment]
@Caleb-Cohen, here we go!

![Diagram-Together](https://user-images.githubusercontent.com/102708144/201696710-680847bd-4b5b-45e8-859d-aca93623ab2d.png)

**Reply by Caleb-Cohen** [reply]
@Natashka333 Can you update the event view modal to contain a RSVP component? Could probably be the exact same as the delete (trash can) where you click the icon and it prompts a "Just this event" or "all events"?

**Comment by Daatari** [comment]
What is the purpose of the Tool Together?

- To explore events that the #100devs Community has to offer.
- To post events to be advertised. 
- To show the the the schedules of events and rsvp to them

**Comment by Natashka333** [comment]
Hey, team! I have created two mock-ups of the landing page. 
The first is just adding navigate to the calendar button to the original design.
The second is totally new design. 

https://www.figma.com/file/mbvX31X6ZQXa4jWlD2qZ40/Together?node-id=0%3A1&t=1FJul50c5PbRC5QY-1

As always, I would like to know your thoughts.


@intelagense, based on your idea, here is

BEFORE
![image](https://user-images.githubusercontent.com/102708144/210040086-93f6197b-5937-4cd9-83b9-b526418e623d.png)


and

 AFTER 
![Screen Shot 2022-12-30 at 12 09 42 AM](https://user-images.githubusercontent.com/102708144/210039995-0f67662e-9866-4591-9727-4fe8e61e7b7d.png)

**Reply by Caleb-Cohen** [reply]
You did such a great job taking @intelagense design and making it your own! Just missing the mayonnaise.

**Comment by Caleb-Cohen** [comment]
@Natashka333. @MatthewBozin and I were talking about the event view modal buttons. What are your thoughts on putting the Closed button on top and the other buttons on the bottom?

Currently we have 4 buttons we need to consider where they go. Close, edit, delete, and RSVP.

**Reply by Natashka333** [reply]
I played with the buttons a little bit. Not sure I like the idea of having some of them on the bottom. 
https://www.figma.com/file/mbvX31X6ZQXa4jWlD2qZ40/Together?node-id=0%3A1&t=B6wTqHiV3Dq2alsh-1

**Reply by Caleb-Cohen** [reply]
This is my favorite. Can you move RSVP to the other buttons? 
<img width="187" alt="Screenshot 2023-01-26 at 12 07 19 AM" src="https://user-images.githubusercontent.com/75268174/214769131-43d2eab5-7c79-45fc-a542-4b933643f264.png">
<img width="262" alt="Screenshot 2023-01-26 at 12 07 42 AM" src="https://user-images.githubusercontent.com/75268174/214769174-7b4628ca-d440-4e41-ada7-844df803050b.png">
